src/js/pages/Arbitration/index.scss

```diff
@@ -13,6 +13,10 @@
   padding: 3px 10px;
 }
 
+.arbitrationDialog button svg {
+  margin-top: 10px !important;
```

Comment:
Can you use `mt-2` instead? (you can change the number too)

Reply:
I tried and the checkbox refused to move :(